README.md

```diff
@@ -203,6 +203,50 @@ the signing key (the loading process will have to be done with other tools and w
 you to provivde your parent key password) into the TPM and referencing its handle in the command 
 you want to call with the credential helper. 
 
+#### Password-Encrypted Private Keys
+You can pass a password-encrypted private key to the credential helper for signing the request. The credential helper supports two formats of PKCS#8 private key files:
+- Unencrypted: `-----BEGIN PRIVATE KEY-----`
+- Password-encrypted: `-----BEGIN ENCRYPTED PRIVATE KEY-----` (using PBES2)
+
+To encrypt an unencrypted private key stored on disk, use a tool of your choice. The following example uses OpenSSL in a Bash environment:
+
+```bash
+openssl pkcs8 -topk8 -in unencrypted-key.pem -out encrypted-key.pem -passout pass:password -v2 aes-256-cbc
+```
+
+This command encrypts a PEM file containing an unencrypted private key in PKCS#8 format using the AES-256-CBC cipher with the password "password". The encrypted key is saved to a PEM file. Supported ciphers include:
+
+- AES-128-CBC
+- AES-192-CBC
+- AES-256-CBC
+
+You can also encrypt the key using a different Pseudorandom Function (PRF):
```

Comment:
Minor, but don't capitalize the first letters in each of the words, "pseudorandom function".